logger.py

```diff
@@ -1,26 +1,26 @@
 import logging
 import os
 
-class Logger:
-    def __init__(self):
-        # Prod mode
-        if not os.getenv("DEV_MODE"):
-            import google.cloud.logging
 
-            client = google.cloud.logging.Client()
-            client.setup_logging()
+def init_logger():
+    # Prod mode
+    if not os.getenv("DEV_MODE"):
```

Comment:
If possible, could we avoid making this if statement and NOT. It can be confusing to read

I'd prefer it if it was either
If Dev mode:
  setup dev logging
else if Prod MOde:
  setup prod mode
else:
  error 

Or


if Prod Mode:
  setup prod logging
else if dev mode
  setup dev logging
else:
  error

Reply:
I didn't want to have a `PROD_MODE` because then, we will have to create a `PROD_MODE` in the cloud function environment. Swapped it around to if Dev else prod